contrib/hack/kubectl

```diff
@@ -34,7 +34,7 @@ if [ -n "${PORT}" ]; then
 	SERVER_INFO="--server localhost:${PORT}"
 fi
 
-docker run --rm -ti --net host \
```

Comment:
Jenkins does not support TTYs and breaks when you run docker commands that provision them. If we're using this script in the testing framework then it's necessary to remove the `-ti`. Most of what we'd do with this image seems to not require input, so it makes sense to me to deal with those specific use cases if they start to become prevalent.

If you feel it's important ASAP, we could add an environment variable flag to this script that specifies whether the command we're running needs a TTY (so you'd call `WITH_TTY=1 kubectl run $ARGS`).